elastic/kibana
# [Cloud Security] Fix for data view error in Console

## Summary

Currently when user goes to Findings page without installing any integration, an error will show up in the console. This PR fixes it

**How to Test**
- Navigate to Findings page without installing any integration
- Open console and make sure there are no error message there
